test/ETHStaticAToken.test.ts

```diff
@@ -73,7 +75,20 @@ describe("ETHStaticAToken", function () {
       governor_address: BigInt(owner.starknetContract.address),
     });
 
-    rewAaveTokenL2 = await rewAaveContractFactory.deploy({
+    rewAaveTokenImplementation = await rewAaveContractFactory.deploy();
```

Comment:
I'd rather keep the chain (L2) in the variable name. Also, I've understood this recently, but a good naming convention for contracts under a proxy seems to be:
- `contractNameImplemantion` for the implementation contract
- `contracNameProxy` for the proxy
- `contractName` for the proxied contract

Reply:
No pb if you don't change it here. I was just sharing this. We could potentially do that in a refactoring PR if we have the time to.

Reply:
Yeah that naming scheme is used [here](https://github.com/aave-starknet-project/aave-starknet-bridge/pull/32/files#diff-c52ab99d41f7be91138f669dc6d12d85892a70aeb50f5ed4229e594c4f20b2ffR74) and I think it makes it a lot more readable. I'll add it as a task for later.

Reply:
I'm also a fan of prefixing everything with l1 and l2

Reply:
https://github.com/aave-starknet-project/aave-starknet-bridge/projects/1#card-81478920